docs/mongocli/command/mongocli-iam-organizations-describe.txt

```diff
@@ -85,7 +85,7 @@ If the command succeeds, the CLI returns output similar to the following sample.
 .. code-block::
 
    ID     NAME
-   <Id>   <Name>
+   <ID>   <Name>
```

Comment:
This reverts a previous change as the Id format is only used by atlas sdk

[see atlas version here](https://github.com/mongodb/mongodb-atlas-cli/blob/6a4bc44c8210b78f10e8d62e01dbb7fad3b00587/docs/atlascli/command/atlas-organizations-describe.txt#L85-L88)